templates/userMatch.interface.hbs

```diff
@@ -0,0 +1,11 @@
+<div class="required-field">
+  <button class="btn" data-remove-field><i class="fa fa-minus fa-fw fa-lg"></i></button>
+  <input name="column" class="form-control" type="text" placeholder="Name"/>
+  <label class="select-proxy-display">
+    <select class="hidden-select form-control" name="required-field-type">
+      <option value="equals">Equals</option>
+    </select>
+    <span class="icon fa fa-chevron-down"></span>
+  </label>
+  <input name="value" class="form-control" type="text" placeholder=""/>
```

Comment:
Is there a reason you left the placeholder blank?

![image](https://user-images.githubusercontent.com/290733/83147895-29acb100-a0f0-11ea-8fed-23b6f4aeb6f0.png)

This doesn't exactly say what the last field is for.

```suggestion
  <input name="value" class="form-control" type="text" placeholder="Value"/>
```